<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SignUp{background-image: url(:/background/images/signin.jpg);}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 120, 84);
font: 24pt &quot;Franklin Gothic Demi Cond&quot;;</string>
   </property>
   <property name="text">
    <string>    Child</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 120, 84);
font: 24pt &quot;Franklin Gothic Demi Cond&quot;;</string>
   </property>
   <property name="text">
    <string>    Parent</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="usernameParentTE">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="passwordParentTE">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="passwordChildTE">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="usernameChildTE">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:  12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:  12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:  12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:  12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font:  12pt &quot;Segoe UI&quot;;
background-color: rgb(199, 59, 36);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
